cluster/saltbase/salt/nginx/init.sls

```diff
@@ -28,10 +30,20 @@ nginx:
     - group: root
     - mode: 644
 
+{% if grains.cloud in ['gce'] %}
+/etc/kubernetes/manifests/nginx.json:
+  file:
+    - managed
+    - source: salt://nginx/nginx.json
+    - user: root
+    - group: root
+    - mode: 644
+{% else %}
 nginx-service:
   service:
     - running
     - name: nginx
+{% endif %}
     - watch:
```

Comment:
actually, the kubelet does pick up changes to the file.  (although I agree that these particular watches don't do anything, moved into the else block)

Added "require" to ensure the sequence we need.